ee/codegen_integration/test_zero_diff_transforms.py

```diff
@@ -0,0 +1,121 @@
+import pytest
+import json
+import os
+from pathlib import Path
+import subprocess
+import tempfile
+from typing import Dict, List, Tuple
+
+from vellum_ee.workflows.display.workflows.base_workflow_display import BaseWorkflowDisplay
+
+
+def _collect_file_map(root: Path) -> Dict[str, str]:
+    """Collect all Python files from a directory tree into a dict mapping relative path to contents."""
+    file_map: Dict[str, str] = {}
+    for path in root.rglob("*.py"):
+        # Skip __pycache__ or other non-source directories
+        if "__pycache__" in path.parts:
+            continue
+        rel_path = str(path.relative_to(root))
+        file_map[rel_path] = path.read_text(encoding="utf-8")
+    return file_map
+
+
+def _compute_diff(
+    original_root: Path,
+    generated_root: Path,
+) -> Tuple[List[str], List[str], List[str]]:
+    """
+    Compare files between original and generated directories.
+
+    Returns:
+        Tuple of (original_only, generated_only, modified) file lists
+    """
+    original_files = _collect_file_map(original_root)
+    generated_files = _collect_file_map(generated_root)
+
+    orig_keys = set(original_files.keys())
+    gen_keys = set(generated_files.keys())
+
+    original_only = sorted(orig_keys - gen_keys)
+    generated_only = sorted(gen_keys - orig_keys)
+
+    modified = sorted(rel for rel in orig_keys & gen_keys if original_files[rel] != generated_files[rel])
```

Comment:
DevinAI for the `modified` files, I want to be able to see what the diff was in the assertion error message below